src/main/java/com/cloudbees/jenkins/support/impl/UpdateCenter.java

```diff
@@ -66,16 +67,9 @@ public void writeTo(OutputStream os) throws IOException {
                                 out.println("Last updated: " + updateCenter.getLastUpdatedString());
                             }
 
-                            out.println("=== Proxy ===");
-                            ProxyServer proxyServer = AHCUtils.getProxyServer();
-                            if (proxyServer != null) {
-                                out.println(" - Host: " + proxyServer.getHost());
-                                out.println(" - Port: " + proxyServer.getPort());
-
-                                out.println(" - No Proxy Hosts: ");
-                                for (String noHost : proxyServer.getNonProxyHosts()) {
-                                    out.println(" * " + noHost);
-                                }
+                            // Only do this part of the async-http-client plugin is installed.
+                            if (Helper.getActiveInstance().getPlugin("async-http-client") != null) {
+                                addProxyInformation(out);
                             }
```

Comment:
You could also output that "async-http-client" is not installed and that is the reason why you are not outputting any proxy info. It could help in a troubleshooting scenario. That could mean moving the if statement into the `addProxyInformation` method to leave it to that method to also print something like 
```
out.println("Proxy: 'async-http-client' not installed so proxy info available.");
```
or something like that.